scan/spec/test_command_generator_spec.rb

```diff
@@ -1,5 +1,8 @@
 describe Scan do
+  require "rspec/mocks/standalone"
```

Comment:
I've also moved it right inside before(:all) block to make its intent more clear.